tests/foreman/api/test_convert2rhel.py

```diff
@@ -251,7 +249,7 @@ def test_convert2rhel_oracle(target_sat, oracle, activation_key_rhel, version):
         },
     )
     # wait for job to complete
-    wait_for_tasks(
+    target_sat.api_factory.wait_for_tasks(
```

Comment:
Wait for task is part of Capsule mixins, not api_factory, right?
```suggestion
    target_sat.wait_for_tasks(
```